styles.css

```diff
@@ -1,504 +1,1092 @@
+    position: relative;
+    right: 20px;
+    top: 20px;
+  }
+
+  .about-para {
+    max-width: 72%;
+    font-family: "Poppins", sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 16px;
+    line-height: 24px;
+    color: #344563;
+    margin: -25px 30px -8px 17%;
+  }
+
+  .about-btn {
+    width: 166px;
+    height: 45px;
+    border: 1px solid #6070ff;
+    border-radius: 8px;
+    font-family: "Poppins", sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 17px;
+    line-height: 24px;
+    letter-spacing: 0.03em;
+    color: #396df2;
+    background-color: #fff;
+    margin-left: 17%;
+    margin-top: 39px;
+  }
+
+  .section2 {
+    width: 60%;
+    padding: 30px 1px;
+    background: #fff;
+    border-top-right-radius: 100px;
+  }
+
+  .section2-ul {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    width: 92%;
+    background-color: #fff;
+    margin-top: 222px;
+    align-self: left;
+  }
+
+  .section2-li {
+    width: 92%;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    margin-top: 5px;
+    margin-bottom: 23px;
+  }
+
+  .section2-h2 {
+    font-family: "Poppins", sans-serif;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 20px;
+    align-items: center;
+    color: #344563;
+  }
+
+  .icon2 {
+    position: relative;
+    top: 23px;
+    right: 28px;
+  }
+
+  .languages {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    width: 54%;
+  }
+
+  .lan {
+    font-family: "Poppins", sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 15px;
+    line-height: 20px;
+    color: #344563;
+    text-align: center;
+    width: 129px;
+    height: 108px;
+    padding-top: 18px;
+    padding-right: -8px;
+    background: #f7f7f9;
+    border-radius: 10px;
+  }
+
+  .hr {
+    border-bottom: 2px solid rgb(188, 188, 188);
+    position: relative;
+    top: 44px;
+    width: 92%;
+  }
+
+  /* footer */
+
+  footer {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    gap: 20px;
+    width: 100%;
+    height: 750px;
+    background-color: #6070ff;
+    border-top-left-radius: 100px;
+    background-image: url(./images/contact\ form\ background\ shapes@2x.png);
+    background-size: 100%;
+    background-position-y: 80px;
+    background-repeat: no-repeat;
+  }
+
+  .contact_paragraph {
+    font-family: "Poppins", sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 28px;
+    text-align: center;
+    width: 540px;
+    color: #ebebff;
+    margin-top: -31px;
+  }
+
+  .contact_title {
+    font-family: "Poppins", sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 35px;
+    line-height: 52px;
+    color: #fff;
+    margin-top: 100px;
+  }
+
+  .form-section {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+  }
```

Comment:
- [OPTIONAL] Great job on implementing the form setion 👍 💯 ... 
- - As I already mentioned above on-screen sizes between `376px` and `769px`, the `form` section of your page does not span fully to the left and right of the screen. please use `Relative width units` like `width: 100%;`, Your portfolio will look great on all screens. Thanks 😄 
- SCREENSHOT FOR REFERENCE:
![image](https://user-images.githubusercontent.com/107632780/229273804-fe3e0686-a1d3-425b-81cb-c20d889c72aa.png)